app/pages/sections/order-items/catalogue-solutions/delete/confirmation/contextCreator.test.js

```diff
@@ -0,0 +1,23 @@
+import manifest from './manifest.json';
+import { getContext } from './contextCreator';
+import { baseUrl } from '../../../../../../config';
+
+describe('delete-catalogue-confirmation contextCreator', () => {
+  describe('getContext', () => {
+      it('should return the title', () => {
+      const orderId = 'order-1';
+      const context = getContext({ orderId });
+      expect(context.title).toEqual('Catalogue order-1 deleted');
+    });
+
+    it('should return the order description title', () => {
+      const context = getContext({});
+      expect(context.orderDescriptionTitle).toEqual(`${manifest.orderDescriptionTitle}`);
+    });
+
+    it('should return the description', () => {
+      const context = getContext({});
+      expect(context.description).toEqual(manifest.description);
+    });
+  });
+});
```

Comment:
There are a couple of linting issues with this file:

```shell
app/pages/sections/order-items/catalogue-solutions/delete/confirmation/contextCreator.test.js
  3:10  error  'baseUrl' is defined but never used           no-unused-vars
  7:1   error  Expected indentation of 4 spaces but found 6  indent
```

Reply:
resolved